### DISCUSSION TITLE
OpenDNS certificate not working

### DISCUSSION THREAD

**Discussion by jameshskoh** [paulmillr/encrypted-dns#162] [Q&A]
I downloaded the OpenDNS profile (tried both signed or unsigned), and my iPad just won't detect the .mobileconfig file as a config to install. I suspect that the updated profile is corrupted.

Reasons:
1. When I open up the file to check myself, I saw some "gibberish" inside.
2. I tried the older version (signed around Nov 23), **that works fine**. The new one, which I can see is committed around Nov 24, has issues.

**Comment by Terabyte1385** [comment]
I don't know if this is still relevant but you can copy/save the contents (the _gibberish_) to a `.mobileconfig` file on your computer and either sync it to iCloud and open it from your iPad, AirDrop it to your iPad, or email it to yourself and open it from there.

If you do try this, I would use the unsigned, as you could break the signature and end up with a corrupted file in the process if you use the signed version.
